content/en/network_monitoring/performance/setup.md

```diff
@@ -166,13 +166,15 @@ If you need to use Network Performance Monitoring on other systems with SELinux
 
 If these utilities do not exist in your distribution, follow the same procedure but using the utilities provided by your distribution instead.
 
-### Windows systems
 
 [1]: /infrastructure/process/?tab=linuxwindows#installation
 [2]: /agent/guide/agent-commands/#restart-the-agent
 [3]: https://github.com/DataDog/datadog-agent/blob/master/cmd/agent/selinux/system_probe_policy.te
 {{% /tab %}}
 {{% tab "Agent (Windows)" %}}
+
+### Windows systems
+
```

Comment:
```suggestion
```

Reply:
What we really want here is no header at all (the tab is the header; see how the other sections are). Removing it altogether.